assignments/objects.js

```diff
@@ -19,26 +19,67 @@ const example = {
 
 // Write your intern objects here:
 
+const mitzi = {
+  "id": 1,
+  "name": 'Mitzi' , 
+  "email":'mmelloy0@psu.edu' , 
+  "gender": 'F' 
+}
+const kennan={
+  "id": 2,
+  "name": 'Kennan' , 
+  "email":'kdiben1@tinypic.com' , 
+  "gender": 'M',
+  speak () {
+    console.log("Hello, my name is Kennan!")
+  }
+}
+const keven = {
+  "id": 3,
+  "name": 'Keven' , 
+  "email":'kmummery2@wikimedia.org' , 
+  "gender": 'M' 
+}
+const gannie={
+  "id": 4,
+  "name": 'Gannie' , 
+  "email":'gmartinson3@illinois.edu' , 
+  "gender": 'M'
+}
+const antonietta={
+  "id": 5,
+  "name": 'Antonietta' , 
+  "email":'adaine5@samsung.com' , 
+  "gender": 'F', 
+  multiplyNums: function (num1, num2) {
+    return num1 * num2;
+  }
+}
+
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
 
+console.log(Object.entries(mitzi)[1]);
+
 // Kennan's ID
+console.log(kennan.id);
 
 // Keven's email
-
+console.log(keven.email);
 // Gannie's name
-
+console.log(gannie.name);
 // Antonietta's Gender
+console.log(antonietta.gender);
 
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
+console.log(kennan.speak());
```

Comment:
```suggestion
kennan.speak = () => {
    return 'Hello, my name is Kennan!';
}
console.log(kennan.speak());
```
here's how you can add a method to an existing object